fms_core/resources.py

```diff
@@ -114,6 +114,21 @@ def import_field(self, field, obj, data, is_m2m=False):
 
         elif field.attribute == 'volume_history':
             obj.volume_history = [create_volume_history("update", data["Volume (uL)"])]
+        # if sample is in tube
+        elif field.attribute == 'container':
+            if data['Container Kind'] == 'tube':
```

Comment:
instead of hard-coding 'tube' can we use `CONTAINER_SPEC_TUBE.container_kind_id`?